exercises/2.md

```diff
@@ -33,9 +33,8 @@ drwxr-xr-x    5 root     root          4096 Mar  2 16:20 lib
 ```
 What happened? Behind the scenes, a lot of stuff happened. When you call `run`,
 1. The Docker client contacts the Docker daemon
-2. The Docker daemon downloads the image (alpine in this case) from Docker Hub
```

Comment:
Ah, right, of course! - we could also include that, the reason it's not pulling the image, is because they've just done it themselves :)